client/src/pages/Login/LoginPage.jsx

```diff
@@ -38,48 +44,72 @@ const LoginPage = () => {
 
       navigate("/dashboard");
     } catch (error) {
-      console.error("Login error");
-      alert(
+      console.error("Login error: ", error);
+      setLoginErrorMessage(
         "There was an error logging in. Please check your credentials and try again."
       );
     }
   };
 
   return (
     <div className="landing-body">
-      <div className="background-blue item-container parent-center">
-        <h1 className="logo">Fitconnect</h1>
+      <div className="item-container parent-center">
+        <h1 className="logo-text">Fitconnect</h1>
+        <p className={loginStyles["subtext"]}>Welcome Back!</p>
         <form className={loginStyles["form"]}>
-          <label className={loginStyles["label"]}>
+          <div className={loginStyles["input-container"]}>
+            <label className={loginStyles["label"]} for="username">
```

Comment:
I'm getting an error on my FE when I open this page:
<img width="1075" alt="image" src="https://github.com/hsu-isaac/fitconnect/assets/85271794/b168803e-fcbf-47db-b592-39c57ab48afe">

I'm not familiar with the 'for' syntax in label elements, would you mind explaining this to me?

Reply:
Huhh that's weird, let me get back to this and take a look at using "for".

I used the "for" atrtibute previously in labels so it attaches itself to an element:
https://www.w3schools.com/tags/tag_label.asp

I'll look into this :)